commit 00e8936bc5af4b6bfdb53126819d50851e79d082 (refs/changes/71/2971/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-11-20T14:25:17-07:00 (1 year, 11 months ago)
    
    joyent/node-manta#336 node-manta affected by usdt cross-arch build bug

diff --git a/CHANGES.md b/CHANGES.md
index 15b9d7d..89a3ce7 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,12 @@
 
 ## not yet released
 
+## 5.1.1
+
+Bring in an update to dtrace-provider by way of restify-clients.
+
+- joyent/node-manta#336 node-manta affected by usdt cross-arch build bug
+
 ## 5.1.0
 
 Minor version bump due to a backwards-compatible addition to the multipart
diff --git a/package.json b/package.json
index d819c40..138fa23 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "type": "git",
         "url": "git://github.com/joyent/node-manta.git"
     },
-    "version": "5.1.0",
+    "version": "5.1.1",
     "main": "./lib/index.js",
     "dependencies": {
         "assert-plus": "^1.0.0",
@@ -27,7 +27,7 @@
         "path-platform": "~0.0.1",
         "progbar": "^1.1.0",
         "readable-stream": "~1.1.9",
-        "restify-clients": "~1.4.1",
+        "restify-clients": "~1.5.2",
         "showdown": "~1.4.2",
         "smartdc-auth": "^2.4.1",
         "strsplit": "1.0.0",
